src/pages/PasswordReset/ChangePassword.jsx

```diff
@@ -0,0 +1,114 @@
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import ChangePasswordActions from
+  '../../actions/resetActions/changePasswordActions';
+import './reset.scss';
```

Comment:
The name of the scss file should match the name of the component

Reply:
this is because this scss file is shared between two files